Dependencies/napi/napi-direct/source/js_native_api_v8.cc

```diff
@@ -3036,10 +3036,16 @@ napi_status napi_detach_arraybuffer(napi_env env, napi_value arraybuffer) {
   v8::Local<v8::ArrayBuffer> it = value.As<v8::ArrayBuffer>();
   RETURN_STATUS_IF_FALSE(
       env, it->IsExternal(), napi_detachable_arraybuffer_expected);
+#if V8_MAJOR_VERSION > 7 || (V8_MAJOR_VERSION == 7 && V8_MINOR_VERSION >= 4)
   RETURN_STATUS_IF_FALSE(
       env, it->IsDetachable(), napi_detachable_arraybuffer_expected);
 
   it->Detach();
+#else
+  RETURN_STATUS_IF_FALSE(
+      env, it->IsNeuterable(), napi_detachable_arraybuffer_expected);
 
+  it->Neuter();
+#endif
```

Comment:
Did you find out why the linux version is using an old version of V8? How does node get away with this?

Reply:
Let me check  with latest Ubuntu. My packages are not latest on my Linux box

Reply:
I've added a note here : https://github.com/BabylonJS/BabylonNative/issues/791